File: problemas/1100/1143.dart
import 'dart:io';
import 'dart:math';

void main() {
  final n = int.parse(stdin.readLineSync());
  for (int i = 1; i <= n; i++) {
    stdout.writeln('$i ${i * i} ${pow(i, 3)}');
  }
}

File: problemas/1100/1143.cs
using System;

namespace uri1143
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 1; i <= n; i++)
            {
                Console.WriteLine($"{i} {i * i} {Math.Pow(i, 3)}");
            }
        }
    }
}